AltBeacon.Library/src/AltBeacon/Beacon/BeaconParser.cs

```diff
@@ -416,8 +417,7 @@ public BeaconParser SetBeaconLayout(string beaconLayout)
 
                 if (!found)
                 {
-                    ////TODO LogManager
-                    ////LogManager.d(Tag, "cannot parse term " + term);
+                    Logger.Debug("cannot parse term " + term);
```

Comment:
Okay, I will change the flag name. So what about the other thing that I told you above ? Does it sounds good or not even necessary ?

Also, what about the comments below ?